--- sdk_source/make/project.mk
+++ sdk_source/make/project.mk.new
@@ -239,10 +239,10 @@
 EXTRA_LDFLAGS ?=
 LDFLAGS ?= -nostdlib \
 	-u call_user_start_cpu0	\
-	$(EXTRA_LDFLAGS) \
 	-Wl,--gc-sections	\
 	-Wl,-static	\
 	-Wl,--start-group	\
+	$(EXTRA_LDFLAGS) \
 	$(COMPONENT_LDFLAGS) \
 	-lgcc \
 	-lstdc++ \
--- sdk_source/components/spi_flash/src/spi_flash_raw.c
+++ sdk_source/components/spi_flash/src/spi_flash_raw.c.new
@@ -33,7 +33,6 @@
     SET_PERI_REG_MASK(PERIPHS_SPI_FLASH_CTRL,SPI_ENABLE_AHB);
     SET_PERI_REG_MASK(CACHE_FLASH_CTRL_REG,CACHE_READ_EN_BIT);
 }
-void Cache_Read_Enable_New(void) __attribute__((alias("Cache_Read_Enable_2")));
 
 uint32_t spi_flash_get_id_raw(esp_spi_flash_chip_t *chip)
 {
--- sdk_source/components/esp8266/source/startup.c
+++ sdk_source/components/esp8266/source/startup.c.new
@@ -41,7 +41,7 @@
 extern int wifi_nvs_init(void);
 extern esp_err_t esp_pthread_init(void);
 
-static void user_init_entry(void *param)
+void user_init_entry(void *param)
 {
     void (**func)(void);
 
