--- esp8266_sdk_source/make/project.mk	2018-08-05 12:09:38.866100747 -0500
+++ esp8266_sdk_source/make/project.mk.new	2018-08-05 12:10:22.517956971 -0500
@@ -239,10 +239,10 @@
 EXTRA_LDFLAGS ?=
 LDFLAGS ?= -nostdlib \
 	-u call_user_start_cpu0	\
-	$(EXTRA_LDFLAGS) \
 	-Wl,--gc-sections	\
 	-Wl,-static	\
 	-Wl,--start-group	\
+	$(EXTRA_LDFLAGS) \
 	$(COMPONENT_LDFLAGS) \
 	-lgcc \
 	-lstdc++ \
--- esp8266_sdk_source/components/spi_flash/src/spi_flash.c	2018-08-14 21:29:48.296617040 -0500
+++ esp8266_sdk_source/components/spi_flash/src/spi_flash.c.new	2018-08-14 21:30:31.842735757 -0500
@@ -234,7 +234,6 @@
     SET_PERI_REG_MASK(PERIPHS_SPI_FLASH_CTRL,SPI_ENABLE_AHB);
     SET_PERI_REG_MASK(CACHE_FLASH_CTRL_REG,CACHE_READ_EN_BIT);
 }
-void Cache_Read_Enable_New(void) __attribute__((alias("Cache_Read_Enable_2")));
 
 static uint32_t IRAM_ATTR spi_flash_get_id(void)
 {
