--- esp8266_sdk_source/make/project.mk	2018-08-05 12:09:38.866100747 -0500
+++ esp8266_sdk_source/make/project.mk.new	2018-08-05 12:10:22.517956971 -0500
@@ -239,10 +239,10 @@
 EXTRA_LDFLAGS ?=
 LDFLAGS ?= -nostdlib \
 	-u call_user_start_cpu0	\
-	$(EXTRA_LDFLAGS) \
 	-Wl,--gc-sections	\
 	-Wl,-static	\
 	-Wl,--start-group	\
+	$(EXTRA_LDFLAGS) \
 	$(COMPONENT_LDFLAGS) \
 	-lgcc \
 	-lstdc++ \
--- esp8266_sdk_source/components/spi_flash/src/spi_flash_raw.c	2018-09-17 20:37:57.664786861 -0500
+++ esp8266_sdk_source/components/spi_flash/src/spi_flash_raw.c.new	2018-09-17 20:37:50.797081790 -0500
@@ -33,7 +33,6 @@
     SET_PERI_REG_MASK(PERIPHS_SPI_FLASH_CTRL,SPI_ENABLE_AHB);
     SET_PERI_REG_MASK(CACHE_FLASH_CTRL_REG,CACHE_READ_EN_BIT);
 }
-void Cache_Read_Enable_New(void) __attribute__((alias("Cache_Read_Enable_2")));
 
 uint32_t spi_flash_get_id_raw(esp_spi_flash_chip_t *chip)
 {
--- esp8266_sdk_source/components/esp8266/source/startup.c	2018-10-05 20:09:03.077295926 -0500
+++ esp8266_sdk_source/components/esp8266/source/startup.c.new	2018-10-05 20:08:53.713816636 -0500
@@ -41,7 +41,7 @@
 extern int wifi_nvs_init(void);
 extern esp_err_t esp_pthread_init(void);
 
-static void user_init_entry(void *param)
+void user_init_entry(void *param)
 {
     void (**func)(void);
 
