--- RF24_source/RF24_config.h	2016-02-14 11:07:26.562422036 -0600
+++ RF24_source/RF24_config.h.new	2016-02-14 11:07:01.691561846 -0600
@@ -116,14 +116,15 @@
 // Arduino DUE is arm and does not include avr/pgmspace
 #if defined (ARDUINO_ARCH_ESP8266)
 
-  #define PSTR(x) (x)
-  #define printf Serial.printf
-  #define sprintf(...) os_sprintf( __VA_ARGS__ )
-  #define printf_P printf
-  #define strlen_P strlen  
-  #define PROGMEM
-  #define pgm_read_byte(addr) (*(const unsigned char *)(addr))
-  #define pgm_read_word(p) (*(p))
+// This stuff is defined in Arduino.h
+//  #define PSTR(x) (x)
+//  #define printf Serial.printf
+//  #define sprintf(...) os_sprintf( __VA_ARGS__ )
+//  #define printf_P printf
+//  #define strlen_P strlen  
+//  #define PROGMEM
+//  #define pgm_read_byte(addr) (*(const unsigned char *)(addr))
+//  #define pgm_read_word(p) (*(p))
   #define PRIPSTR "%s"
 
 #elif defined(ARDUINO) && ! defined(__arm__) && !defined (__ARDUINO_X86__)
